pennylane/decorator.py

```diff
@@ -14,12 +14,25 @@
 """
 .. _qnode_decorator:
 
+Creating QNodes
```

Comment:
Since this section also talks about ML interfaces...it is a bit unclear still what exactly it wants to say. I assume that's due to the difference of code docstrings to a well-ordered didactic flow... 

```suggestion
Creating and using QNodes
```

Reply:
Yeah. The main issue here is that these pages are all autogenerated from docstrings, I'm trying to force some sort of flow out of them that's easier for users to follow...